src/utils/rules.ts

```diff
@@ -240,7 +286,7 @@ const getSpaceValue = scale => propVal => {
     }
 
     // check the theme config for a value, or just use the prop
-    val = scale[val] || val;
+    val = val.startsWith('@') ? scale[val.slice(1)] : val || val;
```

Comment:
That looks confusing 😮 Better to add some brackets here

```suggestion
    val = (val.startsWith('@') ? scale[val.slice(1)] : val) || val;
```

Reply:
Updated!